source/admin_manual/list_of_events.rst

```diff
@@ -821,9 +821,12 @@ These events apply only for connections using the ``connection API``.
           some types of connections, but not for others.
 
 
+.. _event_incomming_conn:
```

Comment:
incoming with one m

Reply:
the ref to it is still event_incomming_conn, causing the build to fail

Reply:
oops, fixed now.